src/nvhttp.cpp

```diff
@@ -267,8 +271,8 @@ namespace nvhttp {
     rtsp_stream::launch_session_t launch_session;
 
     launch_session.host_audio = host_audio;
-    launch_session.gcm_key = util::from_hex<crypto::aes_t>(get_arg(args, "rikey"), true);
-    std::stringstream mode = std::stringstream(get_arg(args, "mode"));
+    launch_session.gcm_key = util::from_hex<crypto::aes_t>(get_arg(args, "rikey", "0"), true);
```

Comment:
`rikey` and `rikeyid` are both required arguments, so you can drop the default value for those.

Reply:
Done!